ee/codegen/src/generators/workflow-sandbox-file.ts

```diff
@@ -26,7 +26,7 @@ export class WorkflowSandboxFile extends BasePersistedFile {
   }
 
   protected getModulePath(): string[] {
-    return [this.workflowContext.moduleName, "sandbox"];
+    return [...this.workflowContext.modulePath.slice(0, -1), "sandbox"];
```

Comment:
I think there's currently a conflation of "workflow context" and "project context" that _might_ be worth separating. but also tricky to do bc we pass projects into subworkflows. ideally, subworkflows are just workflow file/directory, and projects are top level. I plan to sketch out a quick proposal to post to the channel some time tn